p2pv2/server.go

```diff
@@ -92,19 +92,23 @@ func (p *P2PServerV2) Stop() {
 }
 
 // SendMessage send message to peers using given filter strategy
-func (p *P2PServerV2) SendMessage(ctx context.Context, msg *p2pPb.XuperMessage, fs FilterStrategy) error {
-	filter := p.getFilter(fs)
+func (p *P2PServerV2) SendMessage(ctx context.Context, msg *p2pPb.XuperMessage,
+	opts ...MessageOption) error {
+	msgOpts := getMessageOption(opts)
+	filter := p.getFilter(msgOpts.filters)
 	peers, _ := filter.Filter()
 	p.log.Trace("Server SendMessage", "logid", msg.GetHeader().GetLogid(), "msgType", msg.GetHeader().GetType(), "checksum", msg.GetHeader().GetDataCheckSum())
 	return p.node.SendMessage(ctx, msg, peers)
 }
 
 // SendMessageWithResponse send message to peers using given filter strategy, expect response from peers
 // 客户端再使用该方法请求带返回的消息时，最好带上log_id, 否则会导致收消息时收到不匹配的消息而影响后续的处理
-func (p *P2PServerV2) SendMessageWithResponse(ctx context.Context, msg *p2pPb.XuperMessage, fs FilterStrategy, withBreak bool) ([]*p2pPb.XuperMessage,
-	error) {
-	filter := p.getFilter(fs)
+func (p *P2PServerV2) SendMessageWithResponse(ctx context.Context, msg *p2pPb.XuperMessage,
+	opts ...MessageOption) ([]*p2pPb.XuperMessage, error) {
+	msgOpts := getMessageOption(opts)
+	filter := p.getFilter(msgOpts.filters)
 	peers, _ := filter.Filter()
+	withBreak := msgOpts.isBreak
```

Comment:
It seems need to append msgOpts.targetPeerAddrs.

Reply:
no need, targetPeerAddrs is reserved for the quick BFT consensus.

Reply:
I see.